src/java.base/share/classes/jdk/internal/icu/impl/NormalizerImpl.java

```diff
@@ -1999,7 +1999,7 @@ public static String canonicalDecomposeWithSingleQuotation(String string) {
                     back=preBack;
                 }
 
-                // this is where we are right now with all these indicies:
+                // this is where we are right now with all these indices:
```

Comment:
Although these are actual typos, they come from upstream ICU code. Changing them locally would make merging complicated, so please exclude ICU related changes from the PR. I guess fixes in other 3rd party libraries are in the same boat.